core/src/main/java/com/backblaze/b2/client/B2StorageClient.java

```diff
@@ -427,6 +440,25 @@ default B2FileVersion getFileInfo(String fileId) throws B2Exception {
         return getFileInfo(B2GetFileInfoRequest.builder(fileId).build());
     }
 
+    /**
+     * @param request specifies the file whose info to fetch.
+     * @return a B2FileVersion object
+     * @throws B2Exception if there's any trouble.
+     */
+    B2FileVersion getFileInfoByName(B2GetFileInfoByNameRequest request) throws B2Exception;
+
+    /**
+     * Just like getFileInfoByName(request), but you only have to specify the
+     * bucketName and the fileName instead of a request object.
+     *
+     * @param bucketName the name of the bucket you want to download from.
```

Comment:
@param bucketName the name of the bucket containing the file you want info about.

Reply:
done